### DISCUSSION TITLE
Philips 329P9H built-in camera freezing while on video calls

### DISCUSSION THREAD

**Discussion by acousticbiker** [MonitorControl/MonitorControl#708] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Since using M1 builds (version 3 onward), when on a video call (e.g., Teams, Zoom) the built-in camera on my Philips 329P9H monitor freezes intermittently. The only way to unfreeze is to disable video in the app, re-insert camera into monitor (it retracts), re-deploy camera, and re-enable video in app.

I don't know enough technically to determine whether MonitorControl could cause this but I can't think of any other change/cause and the behavior aligns with the timing of the change to the M1-compatible MonitorControl.

### Steps to reproduce

Engage in video call using Teams or Zoom while MonitorControl running

### Expected behavior

No freezing during video calls (which was the case pre-M1 compatibility)

### Environment Information (please complete the following information)

```markdown
- macOS version: 11.6
- Mac model: M1 MacBook Air
- MonitorControl version: 4.0
- Monitor(s): Philips 329P9H
- Apple Silicon/M1 (yes or no): yes
```

**Comment by JoniVR** [comment]
Hi,

When it was working "pre-M1 compatibility", I'm assuming this was also using an Intel machine then?

**Comment by acousticbiker** [comment]
Correct. And, in case helpful, I used Lunar for a period of time when they added M1 compatibility and before MonitorControl did and don't remember the freezing being an issue then.

**Comment by waydabber** [comment]
Hi @acousticbiker - for a while you should try not using MonitorControl (you can try Lunar again) and see if the issue persists. I can't think of any reason why this should happen as the built-in webcam most probably uses USB data while DDC control uses I2C embedded in the DisplayPort communication which happens on separate lanes via USB-C. But the internal workings of the display is an unknown, it might be that somehow DDC commands causes problems internally. Lunar and MonitorControl uses the same commands and basic control method on M1.

**Comment by acousticbiker** [comment]
I did try not using MC and that seemed to fix it

**Comment by waydabber** [comment]
This probably means that the MC is incompatible with the display. You might want to try Lunar and see if it causes the same problem. If not, then I advise you to stick with it, if it does, then it is probably best not to use apps that control the display via DDC if you wish to frequently use the built in webcam. An other option might be to seek Philips about this problem or check if there is a firmware update for the display that fixes the issue.

I'll convert this issue to a discussion, maybe other Philips users will add their experiences.

**Comment by iceman60** [comment]
I will try with my 499p9H at work and give you feedback. This issue is interesting as we use a lot of Philips P line displays in my design studio.

**Reply by waydabber** [reply]
That would be useful, thanks! :)

**Reply by iceman60** [reply]
No problem so far... Tried with a Zoom call. Modified brightness during the call, no freeze.

**Reply by waydabber** [reply]
Nice!
